components/builder-api/src/server/resources/channels.rs

```diff
@@ -31,7 +31,7 @@ use server::error::{Error, Result};
 use server::framework::headers;
 use server::framework::middleware::route_message;
 use server::helpers::{self, Pagination, Target};
-use server::models::channel::{DeleteChannel, ListChannels};
+use server::models::channel::{DeleteChannel, ListChannels, NewChannel};
```

Comment:
Would be more consistent with existing (and desired) terminology to use "CreateChannel" vs. "NewChannel"

Reply:
Probably so. I'll go get my paint brush :joy: